src/shipit_code_coverage/shipit_code_coverage/codecov.py

```diff
@@ -76,21 +75,15 @@ def clone_mozilla_central(self, revision):
                                     self.repo_dir,
                                     purge=True,
                                     sharebase=shared_dir,
-                                    branch=b'tip')
+                                    revision=revision,
+                                    networkattempts=7)
```

Comment:
Cool , i did not know about this option

Reply:
Me neither. For some reason I ended up looking at the robustcheckout source code and noticed it!